### PR TITLE
Fixes #10191,BZ1204923 - fixes setting container cpu shares, sets and memory

### PR DESCRIPTION
**Pull request by adamruzicka**
CPU shares in UI was showing wrong value
CPU set was not being sent to docker because of a typo (cpu_sets vs cpu_set)
Memory needs to be sent to docker as a number of bytes, in db it is stored as a string with unit, it gets parsed and converted to bytes before passing to docker

### CODE REVIEW

File: app/views/containers/show.html.erb [hunk 1]
```diff
@@ -25,7 +25,11 @@
             </tr>
             <tr>
               <td><%= _('CPU shares') %></td>
-              <td><%= @container.in_fog.cores %></td>
+              <td><%= @container.in_fog.cpu_shares %></td>
```
Comment (eLobato) [inline app/views/containers/show.html.erb]: Likely this was just bad fixing some merge conflict, thanks @adamruzicka

**Comment by eLobato** [comment]
@adamruzicka Thank you! This seems to work well, you caught a few mistakes here. I was wondering if you could take away the memory parser from the container.rb class (I don't think it's `Container` specific logic, just a parser), move it perhaps to a service class, and then write a couple of tests for it?

Sorry for the long time to respond!

**Comment by adamruzicka** [comment]
You have a point. I moved it to ```Utilities``` class in ```Service``` module as a static method, I couldn't find a place where it should belong.

**Comment by eLobato** [comment]
@adamruzicka build is failing, at least there are a few rubocop offenses :smile:

**Comment by adamruzicka** [comment]
Will fix it tomorrow morning. The ```Utility``` class i okay?

File: app/models/service/utilities.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module Service
```
Comment (eLobato) [inline app/models/service/utilities.rb]: I think this is the way to go, however place it on `app/services/` instead to follow (unspoken..) [conventions from core](https://github.com/theforeman/foreman/tree/develop/app/services), also 'Utility' (singular)

**Comment by eLobato** [comment]
@adamruzicka Sorry for the few nitpicks, it works well, just minor style issues. Ping me if you see me online (I'm usually there during EST hours) if you want to double-check before pushing. 

Thanks again :raised_hands:

**Comment by eLobato** [comment]
Merged as e88059db3a33445f654b199d803e4364662b7eaf, thanks @adamruzicka !

**Comment by elyezer** [comment]
Will make sure to test this. Thank you.

**Pull request closed without merging**